Regular Expressions/RegExp (Basic).sublime-syntax

```diff
@@ -160,14 +140,54 @@ contexts:
     - match: \)
       scope: meta.group.regexp keyword.control.group.regexp
       pop: 1
-    - include: base-literal
+    - include: ingroup-xmode-on
+    - include: base
+    - include: literals
 
   group-body-extended:
     - meta_content_scope: meta.group.extended.regexp
     - match: \)
       scope: meta.group.extended.regexp keyword.control.group.regexp
       pop: 1
-    - include: base-literal-extended
+    - include: ingroup-xmode-off
+    - include: base-extended
+    - include: literals
+
+  ingroup-xmode-on:
+    # Activates 'x' mode
+    - match: (\()(\?[ims]*x[ixms]*(?:-[ims]+)?)(\))
+      captures:
+        1: keyword.control.group.regexp
```

Comment:
Do we want to scope the whole token `meta.group.regexp`?

Reply:
probably we do, nice spot

Reply:
We can add it to your PR and close this one then in order to avoid conflicts due to squashing. Not worth the trouble.

Reply:
I believe I have just added that in https://github.com/sublimehq/Packages/pull/3057/commits/ea373c93e1e64da8ac09f713e90138d7bafcbcf8